Sources/Anthropic/Service/AnthropicServiceFactory.swift

```diff
@@ -25,7 +25,7 @@ public final class AnthropicServiceFactory {
       basePath: String = "https://api.anthropic.com",
       configuration: URLSessionConfiguration = .default,
       debugEnabled: Bool = false)
-      -> some AnthropicService
+      -> AnthropicService
```

Comment:
This is another change to the core that I touched. You don't need the `some` keyword here because the protocol that our concrete types are conforming to is not a PAT. That is, we don't need to use 'reverse generics' and can instead just rely on vanilla polymorphism through protocols. We could also make this change in SwiftOpenAI